test/ut3_user/expectations/test_expectation_anydata.pks

```diff
@@ -203,5 +203,29 @@ create or replace package test_expectation_anydata is
  
   --%test ( Empty Array not equal array with space )
   procedure arr_empty_nqua_arr_e_unord;  
+  
+  --%test ( Failure of comparing nested objects )
```

Comment:
Can we name those tests better?
Maybe something like:
```
--%test(Correctly reports a diff on failure with nested object)
--%test(Correctly reports a diff on failure with double nested object)
--%test(Reports success when comparing identical nested object)
--%test(Reports success when comparing identical double nested object)
--%test(Correctly reports a diff on failure with nested object list)
```

Reply:
Good point.